service/Service/Service.csproj

```diff
@@ -13,6 +13,8 @@
     <ItemGroup>
         <!-- Microsoft.KernelMemory.Core includes also extensions without external dependencies, e.g. all Azure extensions -->
 
+        <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.22.0" />
```

Comment:
please move this in the block below that contains Swashbuckle.AspNetCore (also note the comment above would not apply to this line)